python/ccf/ledger.py

```diff
@@ -830,8 +833,17 @@ def __init__(
         ledger_files: List[str] = []
         for directory in directories:
             for path in os.listdir(directory):
-                if committed_only and not path.endswith(COMMITTED_FILE_SUFFIX):
+                sanitised_path = path
+                if path.endswith(RECOVERY_FILE_SUFFIX):
```

Comment:
`path.removesuffix(RECOVERY_FILE_SUFFIX)`?

Reply:
I wrote it like that at first but unfortunately `removesuffix` was introduced in Python 3.9.0 and we're still on 3.8.

Reply:
Darn; I'm usually a version behind, not ahead :-)